.gitignore

```diff
@@ -54,3 +54,6 @@ util/java/libtiled-java/target
 
 # Linux perf/debugging
 gmon.out
+
+# Vim swap files
+*.swp
```

Comment:
Please don't add lines for that here. If you want git to ignore vim's swap files, you should add this to your global ignore file (set by `core.excludesFile`, by default usually at `$HOME/.config/git/ignore`).